facebook-commerce.php

```diff
@@ -2933,15 +2933,21 @@ public function get_product_fbid( string $fbid_type, int $wp_id, $woo_product =
 		$fb_retailer_id = WC_Facebookcommerce_Utils::get_fb_retailer_id( $woo_product );
 
 		try {
-			$facebook_ids = $this->facebook_for_woocommerce->get_api()->get_product_facebook_ids(
+			$response = $this->facebook_for_woocommerce->get_api()->get_product_facebook_ids(
 				$this->get_product_catalog_id(),
 				$fb_retailer_id
 			);
 
-			if ( $facebook_ids->id ) {
+			if ( !$response->data || empty( $response->data ) ) {
```

Comment:
if ( !$response->data ) alone will check both conditions. 

```suggestion
			if ( !$response->data ) {
```